src/main/java/com/exasol/glue/reader/ExportQueryGenerator.java

```diff
@@ -0,0 +1,85 @@
+package com.exasol.glue.reader;
+
+import static com.exasol.glue.Constants.*;
+
+import com.exasol.glue.ExasolOptions;
+
+/**
+ * A class that generates an Exasol export query.
+ */
+public final class ExportQueryGenerator {
+    private final ExasolOptions options;
+
+    /**
+     * Creates a new instance of {@link ExportQueryGenerator}.
+     *
+     * @param options an user provided options
+     */
+    public ExportQueryGenerator(final ExasolOptions options) {
+        this.options = options;
+    }
+
+    /**
+     * Generates an Exasol CSV export query.
+     *
+     * @param bucketFolder  a folder in the S3 bucket location
+     * @param numberOfFiles a number of files to export the data
```

Comment:
What does this mean? Will the exporter split the data into n files?

Reply:
Yes. I am going to rephrase it.

How about?
```
@param numberOfFiles the number of files the exported data will be splitted into
```

Reply:
Sounds good 👍